### ISSUE TITLE
Simplify TakagiSugeno defuzzification

### ISSUE THREAD

**Issue by jcrada** [fuzzylite/fuzzylite#68]
TakagiSugeno fuzzy aggregation (formerly referred to as accumulation) and implication (formerly referred to as activation) operators are useless in Matlab for the defuzzification of output variables. However, for design (and experimental) purposes, fuzzylite versions 4.0 and 5.0 have attempted to reuse these fuzzy operators in TakagiSugeno controllers. The result of such a redesign is that the typical defuzzification of TakagiSugeno controllers can be achieved only when setting the implication and aggregation operators to `fl::null`. Otherwise, the defuzzification utilises these fuzzy operators to compute the aggregation and implication with them. 

Mathematically, the typical TakagiSugeno defuzzification is given by $y = \dfrac{\sum_i w_iz_i}{\sum_i w_i}$, where $w_i$ is the activation degree of term $i$, and $z_i = \mu_i(w_i)$. With the fuzzy implication and aggregation operators, the typical sum and multiplication are performed utilising the respective implication and aggregation operators. 

Hence, when Matlab's examples arbitrary select any implication and aggregation operators (producing the same results because these are *not* used), fuzzylite actually uses these operators and hence produces different results.

In summary, to achieve the typical defuzzification of TakagiSugeno, it is necessary to set the implication and aggregation operators to `fl::null`. 

It remains to be decided whether this operation in fuzzylite is useful. Any comments on this are more than welcome.

Some information on the topic can be found at:

http://fuzzylite.github.io/fuzzylite/d7/dde/classfl_1_1_weighted_average.html

http://www.fuzzylite.com/cpp/#Weighted_Defuzzifiers
